components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_IMPL/attestation_bootloader_data.c

```diff
@@ -18,6 +18,7 @@
 
 #include "attestation_bootloader_data.h"
 
+#if (DUMMY_ATTESTATION == 1)
```

Comment:
is `DUMMY_ATTESTATION` macro expected to have other numerical values?
if not - consider changing to:
```suggestion
#if defined(DUMMY_ATTESTATION)
```

Reply:
Always defined, comes from mbed_lib.json